tests/test_user_settings.py

```
@@ -0,0 +1,666 @@
+        result = greet(name="Alice")
+        assert result == "Hello, Alice!"
+
+    def test_decorator_logs_warning_on_error(self):
+        """Test that decorator logs a warning when OperationalError occurs."""
+
+        @with_db_fallback(fallback_value=None, log_message="Custom error message")
+        def failing_function():
+            raise pw.OperationalError("DB error")
+
+        with patch("hub_adapter.user_settings.logger") as mock_logger:
+            result = failing_function()
```

Ruff (0.15.2)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)